src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
     }
+
+    // Intentional errors added below
```

Comment:
5. Error 5: The method named `Subtract` violates the standard Java naming conventions by starting with an uppercase letter. Method names should start with a lowercase letter.